recipes/pyeasyfuse/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pyeasyfuse" %}
+{% set version = "2.0.3" %}
+{% set sha256 = "10bc6159d2da2f21f5e6a2bd3a04f9d1bc320e5a8259ce72f95b3cd3de58482d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyeasyfuse-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --use-pep517 --no-deps -vvv
+  entry_points:
+    - easy-fuse=pyeasyfuse.command_line:easy_fuse_cli
+
+requirements:
+  host:
+    - python >=3.7,<3.8
+    - pip
+    - poetry-core
+  run:
+    - python >=3.7,<3.8
+    - numpy 1.21.*
+    - pandas >=1.0.0
+    - python-xxhash 1.4.*
+    - biopython 1.73.*
+    - pysam >=0.15.3
+    - gffutils 0.10.*
+    - bx-python 0.8.*
+    - logzero 1.7.*
+    - pytz 2022.7.*
+    - importlib-metadata
+    - r-base 4.2.*
+    - r-optparse
+    - r-dplyr
+    - r-tidyr
+    - r-tidyselect
+    - r-readr
+    - r-stringr
+    - r-xml
+    - r-randomforest
```

Comment:
As far as I know all R dependencies have to also be specified in the host environment for a recipe to work properly.

Reply:
I was not aware, thanks for informating me! I have added the r-base dependencies to the host section in my next commit.